src/components/Avatar.js

```diff
@@ -26,10 +31,10 @@ class Avatar extends PureComponent {
             <Image
                 ref={el => this.image = el}
                 source={{uri: this.props.source}}
-                style={[
-                    styles.avatarNormal,
-                    ...this.props.style,
-                ]}
+                style={_.union([
```

Comment:
Why `_.union()`?

Reply:
Somebody suggested me in a recent review so I am following that. In real, styles should be automatically merged and duplicate properties should be removed automatically.

Reply:
Reverting it.